sky/data/storage.py

```diff
@@ -1368,24 +1368,22 @@ def _create_s3_bucket(self,
         """
         s3_client = self.client
         try:
-            if region is None:
-                s3_client.create_bucket(Bucket=bucket_name)
-            else:
-                if region == 'us-east-1':
-                    # If default us-east-1 region is used, the
-                    # LocationConstraint must not be specified.
-                    # https://stackoverflow.com/a/51912090
-                    s3_client.create_bucket(Bucket=bucket_name)
-                else:
-                    location = {'LocationConstraint': region}
-                    s3_client.create_bucket(Bucket=bucket_name,
-                                            CreateBucketConfiguration=location)
-                logger.info(f'Created S3 bucket {bucket_name} in {region}')
+            create_bucket_config: Dict[str, Any] = {'Bucket': bucket_name}
+            # If default us-east-1 region of create_bucket API is used,
+            # the LocationConstraint must not be specified.
+            # Reference: https://stackoverflow.com/a/51912090
+            if region and region != 'us-east-1':
```

Comment:
```suggestion
            if region is not None and region != 'us-east-1':
```
Always use if foo is None: (or is not None) to check for a None value.

https://google.github.io/styleguide/pyguide.html#2144-decision